packages/php/src/config.dist.php

```diff
@@ -45,6 +45,16 @@
      * Note that this does not support dot-notation, so only top-level keys can
      * be whitelisted.
      */
+    'allowlist' => [],
+
+    /**
+     * Deprecated, prefer denylist
+     */
+    'blacklist' => [],
+
+    /**
+     * Deprecated, prefer allowlist
+     */
     'whitelist' => [],
```

Comment:
Since this is for the config dist for new installs, we can remove `blacklist` and `whitelist` from it.

```suggestion
```